contrib/mod_sftp/fxp.c

```diff
@@ -385,72 +385,75 @@ static cmd_rec *fxp_cmd_alloc(pool *p, const char *name, char *arg) {
 }
 
 static const char *fxp_strerror(uint32_t status) {
-    switch (status) {
-      case SSH2_FX_OK:
-        return "OK";
```

Comment:
Most of this is indentation/whitespace; I don't know why the existing code was indented two spaces too far.